common/source/docs/common-makeflyeasy-PixSurveyA2.rst

```diff
@@ -0,0 +1,370 @@
+.. _common-makeflyeasy-PixSurveyA2:
```

Comment:
Is this new? It seems odd for this file to have been introduced to the wiki as part of this Sub-focused PR.